libraries/libbase/source/org/pentaho/reporting/libraries/base/config/metadata/ConfigurationMetaDataParser.java

```diff
@@ -45,6 +46,8 @@
   private static Document parseInputStream( final InputStream instream )
```

Comment:
Suggest this goes to public static so that everyone else has one place to call to set up a secure parser.